ogre2/src/Ogre2DepthCamera.cc

```diff
@@ -589,119 +589,124 @@ void Ogre2DepthCamera::CreateDepthTexture()
         ogreCompMgr->addNodeDefinition(baseNodeDefName);
     Ogre::TextureDefinitionBase::TextureDefinition *rt0TexDef =
         baseNodeDef->addTextureDefinition("rt0");
-    rt0TexDef->textureType = Ogre::TEX_TYPE_2D;
+    rt0TexDef->textureType = Ogre::TextureTypes::Type2D;
     rt0TexDef->width = 0;
     rt0TexDef->height = 0;
-    rt0TexDef->depth = 1;
+    rt0TexDef->depthOrSlices = 1;
     rt0TexDef->numMipmaps = 0;
     rt0TexDef->widthFactor = 1;
     rt0TexDef->heightFactor = 1;
-    rt0TexDef->formatList = {Ogre::PF_FLOAT32_RGBA};
-    rt0TexDef->fsaa = 0;
-    rt0TexDef->uav = false;
-    rt0TexDef->automipmaps = false;
-    rt0TexDef->hwGammaWrite = Ogre::TextureDefinitionBase::BoolFalse;
+    rt0TexDef->format = Ogre::PFG_RGBA32_FLOAT;
+    rt0TexDef->textureFlags |= !Ogre::TextureFlags::Uav;
```

Comment:
there is a compile warning here and other places with this line. I think it's for removing the flag? i.e. `rt0TexDef->textureFlags &= ~Ogre::TextureFlags::Uav;`? or maybe we can just set it to `Ogre::TextureFlags::RenderToTexture`